__tests__/test_part1.js

```diff
@@ -7,7 +7,7 @@ beforeAll(async () => {
   page.on('console', (msg) => {
     output += `${msg.text()}\n`;
   });
-  await page.goto('http://localhost:8080/exercises/part1-fizzbuzz/');
+  await page.goto('http://10.103.82.200:8081/exercises/part1-fizzbuzz/');
```

Comment:
I recommend against changing the host and port for these tests; the test server on GitHub is going to be started on port 8080. Should probably change this back in the other _\_\_tests\_\__ files too.

```suggestion
  await page.goto('http://localhost:8080/exercises/part1-fizzbuzz/');
```